The default value of basepython is not compatible with our tox usage.

--- hypothesis-hypothesis-python-6.151.11/hypothesis-python/tox.ini.orig
+++ hypothesis-hypothesis-python-6.151.11/hypothesis-python/tox.ini
@@ -3,7 +3,6 @@
 toxworkdir={env:TOX_WORK_DIR:.tox}
 
 [testenv]
-basepython={env:TOX_PYTHON_VERSION:python3}
 deps =
     -r../requirements/test.in
 extras =
